src/locale/ids/fr.ts

```diff
@@ -116,6 +109,9 @@ export const fr: Locale = {
   'ifsBrowser.doSearchInStreamfiles.progressMessage': `'{0}' dans {1}.`,
   'ifsBrowser.doSearchInStreamfiles.noResults': `Aucun résultat trouvé pour '{0}' dans {1}.`,
   'ifsBrowser.doSearchInStreamfiles.errorMessage': `Erreur lors de la recherche de fichiers.`,
+  'ifsBrowser.file.tooltip.size': `Taille:\t\t\t{0}`,
```

Comment:
Remove a tab to align
```suggestion
  'ifsBrowser.file.tooltip.size': `Taille:\t\t{0}`,
```

Reply:
The current number of tabs makes the columns align - because the text length is different:

![image](https://github.com/codefori/vscode-ibmi/assets/13275072/5904b0aa-6316-4b61-ae01-dafab653bf5c)

Is that not what you get in your instance of VS Code?

Reply:
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/03ba7787-6013-4066-9ba5-39d2d49c6b90)
Sad, right? 😅